subgraph.yaml

```diff
@@ -499,38 +481,3 @@ dataSources:
           handler: handleTransfer
         - event: NewMarketInterestRateModel(address,address)
           handler: handleNewMarketInterestRateModel
-  - kind: ethereum/contract
```

Comment:
why did we change the positioning of comptroller to the end of the file from the beginning?

Reply:
moved it so that comptroller is at top similar to how factory is at top in dex subgraph. resolved. moved it back to the bottom